readthedocs/templates/builds/build_detail.html

```diff
@@ -9,8 +9,9 @@
   <script type="text/javascript" src="{% static 'vendor/knockout.js' %}"></script>
   <script type="text/javascript" src="{% static 'builds/js/detail.js' %}"></script>
   <script type="text/javascript">
+var detail = require('detail');
```

Comment:
Addressed in https://github.com/rtfd/readthedocs.org/commit/26f27ddd77ac745ec060280ae221fdd83bcba8ce